Sources/PointFree/Account/Invite.swift

```diff
@@ -122,6 +122,32 @@ let acceptInviteMiddleware: Middleware<StatusLineOpen, ResponseEnded, Tuple2<Tea
         .flatMap(const(conn |> redirect(to: path(to: .account(.index)))))
 }
 
+let addTeammateViaInviteMiddleware: Middleware<
```

Comment:
this is the middleware that updates the subscription to increase the quantity by 1 and then invites the teammate